src/pages/staticPages/webAPI/WebAPIPage.tsx

```diff
@@ -1,12 +1,86 @@
 import * as React from 'react';
 import {observer} from 'mobx-react';
+import {observable} from "mobx";
 import {PageLayout} from "../../../shared/components/PageLayout/PageLayout";
+import StaticContent from "../../../shared/components/staticContent/StaticContent";
+import internalClient from "shared/api/cbioportalInternalClientInstance";
 import './styles.scss';
 import Helmet from "react-helmet";
 import { Link } from 'react-router';
+import AppConfig from "appConfig";
+import {isNullOrUndefined} from "util";
+import fileDownload from 'react-file-download';
+import {AppStore} from "../../../AppStore";
+import LoadingIndicator from "../../../shared/components/loadingIndicator/LoadingIndicator";
+import getBrowserWindow from "../../../public-lib/lib/getBrowserWindow";
+
+export class UserDataAccessToken {
+    @observable token : string;
+    @observable creationDate: string;
+    @observable expirationDate: string;
+    @observable username: string;
+    constructor(token: string, creationDate: string, expirationDate: string, username: string) {
+        this.token = token;
+        this.creationDate = creationDate;
+        this.expirationDate = expirationDate;
+        this.username = username;
+    }
+}
+
+function buildDataAccessTokenFileContents(dat:UserDataAccessToken | undefined) {
+    if (!isNullOrUndefined(dat)) {
+        const fileContents = "token: " + dat!.token + "\n" +
```

Comment:
just fyi, you can use new es6 string templates to compose multiline strings with placeholders.

Reply:
done